src/plugins/template/src/config.cpp

```diff
@@ -83,6 +83,50 @@ Configuration::Configuration(const ov::AnyMap& config, const Configuration& defa
             log_level = value.as<ov::log::Level>();
         } else if (ov::hint::model_priority == key) {
             model_priority = value.as<ov::hint::Priority>();
+        } else if (ov::optimal_batch_size == key) {
+            try {
+                int tmp_i = value.as<int>();
```

Comment:
we can use unsigned int as `size_t` here instead of `int`

Reply:
We should apply the same data type with property definition, for example:
https://github.com/openvinotoolkit/openvino/blob/f55e967dd5a65d2f7f48f8e1197f20f824b7fe28/src/inference/include/openvino/runtime/properties.hpp#L705

Reply:
For newly added properties, I updated them with type defined in [openvino/src/inference/include/openvino/runtime/properties.hpp](https://github.com/openvinotoolkit/openvino/blob/f55e967dd5a65d2f7f48f8e1197f20f824b7fe28/src/inference/include/openvino/runtime/properties.hpp#L705)